.sage/main.go

```diff
@@ -24,6 +24,13 @@ func main() {
 	)
 }
 
+func DoubleGitCheck(ctx context.Context) error {
+	sg.SerialDeps(ctx, GitVerifyNoDiff)
+	sg.SerialDeps(ctx, GoFormat)
+	sg.SerialDeps(ctx, GitVerifyNoDiff) // this should report a change
```

Comment:
Unexpectedly this call does not return an error, even though the previous format changed a file